samples/unity-of-bugs/Packages/manifest.json

```diff
@@ -1,7 +1,8 @@
 {
   "dependencies": {
-    "com.unity.ide.rider": "1.1.4",
-    "com.unity.test-framework": "1.1.24",
+    "com.unity.ide.rider": "1.2.1",
+    "com.unity.ide.visualstudio": "2.0.11",
```

Comment:
should updating unity also add an extra entry `com.unity.ide.visualstudio`?

Reply:
Fair point! Unity-Package cleanup: https://github.com/getsentry/sentry-unity/pull/351 (has been on my "want to do" for ages!)